### PR TITLE
AD-1: Display location of restaurants

### PR DESCRIPTION
**Pull request by ademdc**

### CODE REVIEW

File: ember/app/templates/restaurant.hbs [hunk 1]
```diff
@@ -68,6 +68,9 @@
               {{/unless}}
             {{/if}}
           </div>
+
+          {{google-map markerLat=model.restaurant.latitude markerLng=model.restaurant.longitude   }}
+          {{log model markerLat=model.restaurant.latitude markerLng=model.restaurant.longitude }}
```
Comment (ZaidABH) [inline ember/app/templates/restaurant.hbs]: Remove `log` statement.

Comment (ZaidABH) [inline ember/app/templates/restaurant.hbs]: Remove `log` statement.

File: ember/app/components/google-map.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default Ember.Component.extend({
   renderGoogleMap(container) {
     let options = {
       center: new window.google.maps.LatLng(43.9, 18),
-      zoom: 7,
+      zoom: 100
```
Comment (ZaidABH) [inline ember/app/components/google-map.js]: Try to play around the `zoom` level so it displays only the map of Sarajevo.

**Comment by ZaidABH** [comment]
Remove the `package-lock.json` file from this PR.

**Review by ZaidABH — CHANGES_REQUESTED** [review]
Remove the package-lock.json file from this PR and fix the things mentioned in the PR comments.

File: ember/app/templates/restaurant.hbs [hunk 3]
```diff
@@ -68,6 +68,9 @@
               {{/unless}}
             {{/if}}
           </div>
+
+          {{google-map markerLat=model.restaurant.latitude markerLng=model.restaurant.longitude   }}
```
Comment (ZaidABH) [inline ember/app/templates/restaurant.hbs]: This looks fine, just remove the extra space at the end.

**Comment by ademdc** [comment]
This `package-lock.json` file was added in the first commit, before I started writing any code. I will try to remove it somehow.

**Pull request closed without merging**